storage/plugins/book-club/src/Repo.php

```
@@ -81,6 +82,153 @@ private function exec(string $sql, string $types = '', array $params = []): bool
+            $name = mb_substr($name, 0, 255, 'UTF-8');
+            $key = mb_strtolower($name, 'UTF-8');
+            $names[$key] = $name;
+        }
+        return array_values($names);
+    }
+
+    private function findOrCreateAuthor(string $name): int
+    {
+        $stmt = $this->db->prepare('SELECT id FROM autori WHERE nome = ? ORDER BY id ASC LIMIT 1');
+        if ($stmt === false) {
+            throw new \RuntimeException('author lookup prepare failed');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '135', column '23'). (undefined)

(MissingImport)